src/job_cache/daemon_cache.cpp

```diff
@@ -72,6 +72,29 @@ static void initialize_logging() {
   }
   wcl::log::subscribe(std::make_unique<JsonSubscriber>(std::move(*log_file_res)));
 
+  const char *bulk_dir = getenv("WAKE_BULK_LOGGING_DIR");
```

Comment:
Not blocking on this, but I'd really like everything like this eventually get moved into WakeConfig. Would be nice to have a singular thing to audit when trying to figure out what goes into wake outside of the wake files

Reply:
ok done!